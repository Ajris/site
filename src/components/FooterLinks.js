import React, {Component} from 'react';
import linkedIn from "../assets/icons/linkedin_icon.png";
import github from "../assets/icons/github_icon.png";
import youtube from "../assets/icons/youtube_icon.png";
import twitter from "../assets/icons/twitter_icon.png";
import mail from "../assets/icons/mail_icon.png";

const Layout = require('antd').Layout;
const Footer = Layout;

class FooterLinks extends Component {
    render() {
        return (
            <Footer>
                <nav className="navbar navbar-expand-sm bg-dark" style={{width: '100%'}}>
                    <div className="align-items-center mx-auto">
                        <ul className="navbar-nav">
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" href="https://linkedin.com/">
                                    <img src={linkedIn} alt="LinkedIn" height={25}/>
                                </a>
                            </li>
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" href="https://github.com/">
                                    <img src={github} alt="Github" height={25}/>
                                </a>
                            </li>
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" href="https://youtube.com/">
                                    <img src={youtube} alt="YouTube" height={25}/>
                                </a>
                            </li>
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" href="https://twitter.com/">
                                    <img src={twitter} alt="Twitter" height={25}/>
                                </a>
                            </li>
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" href="https://gmail.com/">
                                    <img src={mail} alt="Mail" height={25}/>
                                </a>
                            </li>
                        </ul>
                    </div>
                </nav>
            </Footer>
        )
    }
}

export default FooterLinks;