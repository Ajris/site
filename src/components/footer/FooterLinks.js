import React from 'react';
import linkedIn from "../../assets/icons/linkedin_icon.png";
import github from "../../assets/icons/github_icon.png";
import youtube from "../../assets/icons/youtube_icon.png";
import twitter from "../../assets/icons/twitter_icon.png";
import mail from "../../assets/icons/mail_icon.png";

const Layout = require('antd/lib/index').Layout;
const Footer = Layout;

class FooterLinks extends React.Component {
    render() {
        return (
            <Footer>
                <nav className="navbar navbar-expand-sm bg-dark" style={{
                    width: '100%', position: 'fixed', bottom: 0, left: 0,
                    right: 0, height: '40px'}}>
                    <div className="align-items-center mx-auto">
                        <ul className="navbar-nav">
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" rel="noopener noreferrer"
                                   href="https://www.linkedin.com/in/hubert-bratek-762120162/">
                                    <img src={linkedIn} alt="LinkedIn" height={25}/>
                                </a>
                            </li>
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" rel="noopener noreferrer" href="https://github.com/Ajris">
                                    <img src={github} alt="Github" height={25}/>
                                </a>
                            </li>
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" rel="noopener noreferrer"
                                   href="https://www.youtube.com/channel/UCz7j6ER42OyLulY83qn7yRg">
                                    <img src={youtube} alt="YouTube" height={25}/>
                                </a>
                            </li>
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/Ajris4">
                                    <img src={twitter} alt="Twitter" height={25}/>
                                </a>
                            </li>
                            <li className="nav-item py-2 px-3">
                                <a target="_blank" rel="noopener noreferrer" href="mailto:ajrisit@gmail">
                                    <img src={mail} alt="Mail" height={25}/>
                                </a>
                            </li>
                        </ul>
                    </div>
                </nav>
            </Footer>
        )
    }
}

export default FooterLinks;