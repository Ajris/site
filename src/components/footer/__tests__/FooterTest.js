import React from "react";
import { create } from "react-test-renderer";
import FooterLinks from "../FooterLinks";

describe("Login component", () => {
    test("it matches the snapshot", () => {
        const component = create(<FooterLinks/>);
        expect(component.toJSON()).toMatchSnapshot();
    });
});