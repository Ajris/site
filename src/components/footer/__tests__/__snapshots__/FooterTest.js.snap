// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component it matches the snapshot 1`] = `
<section
  className="ant-layout"
>
  <nav
    className="footer navbar navbar-expand bg-dark"
  >
    <div
      className="align-items-center mx-auto"
    >
      <ul
        className="navbar-nav"
      >
        <li
          className="nav-item py-2 px-3"
        >
          <a
            href="https://www.linkedin.com/in/hubert-bratek-762120162/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="LinkedIn"
              height={25}
              src="linkedin_icon.png"
            />
          </a>
        </li>
        <li
          className="nav-item py-2 px-3"
        >
          <a
            href="https://github.com/Ajris"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Github"
              height={25}
              src="github_icon.png"
            />
          </a>
        </li>
        <li
          className="nav-item py-2 px-3"
        >
          <a
            href="https://www.youtube.com/channel/UCz7j6ER42OyLulY83qn7yRg"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="YouTube"
              height={25}
              src="youtube_icon.png"
            />
          </a>
        </li>
        <li
          className="nav-item py-2 px-3"
        >
          <a
            href="https://twitter.com/Ajris4"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Twitter"
              height={25}
              src="twitter_icon.png"
            />
          </a>
        </li>
        <li
          className="nav-item py-2 px-3"
        >
          <a
            href="mailto:ajrisit@gmail"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Mail"
              height={25}
              src="mail_icon.png"
            />
          </a>
        </li>
      </ul>
    </div>
  </nav>
</section>
`;
